src/assets/_project/_blocks/utils/qg-load-google-api.js

```diff
@@ -18,8 +36,9 @@
       }
     };
     if ($('#googleapi').length <= 0) {
+      let googleApiKey = window.qg.franchise.name ? window.qg.franchise.apiKey : window.qg.googleKey;
```

Comment:
should it not be window.qg.franchise.apiKey !== '' then use window.qg.franchise.apiKey else use default key?

do we have a way to track initialisation of maps on the default key and ping google 360 on its usage.

what about primary domains and swapping out the key uat environments.

Reply:
@william-dutton-dsiti 
I have added this  let googleApiKey = window.qg.franchise && window.qg.franchise.apiKey ? window.qg.franchise.apiKey : window.qg.googleKey;

We already have a domian swapping logic in qg-env file , but to make it clear/simpler I have transferred all in the same file

 window.qg.googleKey = window.location.hostname.search(/\bdev\b|\btest\b|\blocalhost\b|\buat\b/) !== -1 ? keys.defGoogle.uat : keys.defGoogle.prod;
  window.qg.googleRecaptchaApiKey = window.location.hostname.search(/\bdev\b|\btest\b|\blocalhost\b|\buat\b/) !== -1 ? keys.defGoogleRecaptcha.uat : keys.defGoogleRecaptcha.prod;

Reply:
what about franshcies in the lower environments? will they use the production keys?

Reply:
I have been told that there is only one in case of Franchise .
I will check with Nirvikar